config/settings.yaml.example

```diff
@@ -66,6 +66,23 @@
 #  :dynflow:
 #    :enabled: true
 
+:telemetry:
+  # one of 'prometheus', 'statsd', 'rails' or 'off'.
```

Comment:
Should this be onesnd only one? I can imagine wanting to enable logger as well as prometheus at the same time

Reply:
Not sure what you mean, but rails log is not very useful, try it yourself. It will be used in very rare situations, otherwise it floods logs.

Reply:
Still, seems to me like unnecessary limitation: I can imagine a case, where the foreman is configured in the infrastructure to use one provider (let's say `prometheus`) and we would ask the user to turn on `statsd` because we could than process further data.

I'm not sure if this will be the case or not, but if I have two options, where one is more strong, and I don't see a reason why we should not do that, the multi-provider option wins for me and it's much easier to do now, than few years from now, when the request arises.

Reply:
I did not get your initial comment, indeed that's a good idea and config is even more readable after the change.